pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -664,6 +664,28 @@ public static boolean haveMinimumJVMVersion(String version) {
     return (jvm.compareTo(version) >= 0);
   }
 
+  @Deprecated
+  public static boolean isProtocolVersion(Connection con, int version) {
```

Comment:
This has just been removed (see https://github.com/pgjdbc/pgjdbc/commit/0221f930b35a6f24e539ac4740886fafaebcaeac ), please do not resurrect it.